xml/System/ReadOnlySpan`1.xml

```diff
@@ -29,9 +29,13 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <typeparam name="T">To be added.</typeparam>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <typeparam name="T">The type of items in the <see cref="System.ReadOnlySpan`1" />.</typeparam>
+    <summary>Provides a type-safe and memory-safe read-only representation of a contiguous region of arbitrary memory.</summary>
+    <remarks>
+      <format type="text/markdown"><![CDATA[`ReadOnlySpan<T>` is a [ref struct](~/docs/csharp/reference-semantics-with-value-types.md#ref-struct-type) that is allocated on the stack and can never escape to the managed heap. Ref struct types have a number of restrictions to ensure that they cannot be promoted to the managed heap, including that they can't be boxed, captured in lambda expressions, assigned to variables of type <xref:System.Object>, assigned to `dynamic` variables, and they cannot implement any interface type.
+
+A `ReadOnlySpan<T>` instance is often used to hold the elements of an array or a portion of an array. Unlike an array, however, a `ReadOnlySpan<T>` instance can point to managed memory, native memory, or memory managed on the stack.]]></format>
```

Comment:
Nit: "to hold" suggests to me that it has its own storage... rather, then span is just referencing other storage.